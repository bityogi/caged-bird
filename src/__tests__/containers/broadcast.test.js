jest.mock('util/axiosClient');

import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
import { MemoryRouter } from 'react-router';

import Broadcast from 'containers/transactions';
import store from 'store';

describe('Login', () => {
  it('renders correctly', () => {
    const rendered = renderer.create(
      <Provider store={store}>
        <MuiThemeProvider>
          <MemoryRouter initialEntries={['/', '/landing']}>
            <Broadcast />
          </MemoryRouter>
        </MuiThemeProvider>
      </Provider>

    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });

});
