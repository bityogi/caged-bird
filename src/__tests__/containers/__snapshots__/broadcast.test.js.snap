// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders correctly 1`] = `
<div
  style={
    Object {
      "WebkitTapHighlightColor": "rgba(0,0,0,0)",
      "backgroundColor": "#ffffff",
      "borderRadius": 2,
      "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
             0 1px 4px rgba(0, 0, 0, 0.12)",
      "boxSizing": "border-box",
      "color": "rgba(0, 0, 0, 0.87)",
      "fontFamily": "Roboto, sans-serif",
      "muiPrepared": true,
      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
    }
  }
>
  <div
    style={
      Object {
        "muiPrepared": true,
        "padding": 16,
        "position": "relative",
      }
    }
  >
    <span
      style={
        Object {
          "color": "rgba(0, 0, 0, 0.87)",
          "display": "block",
          "fontSize": 24,
          "lineHeight": "36px",
          "muiPrepared": true,
        }
      }
    >
      Pending Transactions
    </span>
    <span
      style={
        Object {
          "color": "rgba(0, 0, 0, 0.54)",
          "display": "block",
          "fontSize": 14,
          "muiPrepared": true,
        }
      }
    />
  </div>
  <div
    style={
      Object {
        "height": "auto",
        "muiPrepared": true,
        "overflow": "auto",
      }
    }
  >
    <div
      style={
        Object {
          "muiPrepared": true,
        }
      }
    >
      <table
        className={undefined}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "borderCollapse": "collapse",
            "borderSpacing": 0,
            "fontFamily": "Roboto, sans-serif",
            "tableLayout": "fixed",
            "width": "100%",
          }
        }
      >
        <thead
          className={undefined}
          style={
            Object {
              "borderBottom": "1px solid #e0e0e0",
              "muiPrepared": true,
            }
          }
        >
          <tr
            className={undefined}
            style={
              Object {
                "borderBottom": "1px solid #e0e0e0",
                "color": "rgba(0, 0, 0, 0.87)",
                "height": 48,
                "muiPrepared": true,
              }
            }
          >
            <th
              className={undefined}
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              style={
                Object {
                  "backgroundColor": "inherit",
                  "color": "#9e9e9e",
                  "fontSize": "1.1em",
                  "fontWeight": "normal",
                  "height": 56,
                  "muiPrepared": true,
                  "paddingLeft": 24,
                  "paddingRight": 24,
                  "position": "relative",
                  "textAlign": "left",
                  "textOverflow": "ellipsis",
                  "whiteSpace": "nowrap",
                  "width": "20%",
                }
              }
            >
              Client
            </th>
            <th
              className={undefined}
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              style={
                Object {
                  "backgroundColor": "inherit",
                  "color": "#9e9e9e",
                  "fontSize": "1.1em",
                  "fontWeight": "normal",
                  "height": 56,
                  "muiPrepared": true,
                  "paddingLeft": 24,
                  "paddingRight": 24,
                  "position": "relative",
                  "textAlign": "left",
                  "textOverflow": "ellipsis",
                  "whiteSpace": "nowrap",
                  "width": "15%",
                }
              }
            >
              Coin
            </th>
            <th
              className={undefined}
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              style={
                Object {
                  "backgroundColor": "inherit",
                  "color": "#9e9e9e",
                  "fontSize": "1.1em",
                  "fontWeight": "normal",
                  "height": 56,
                  "muiPrepared": true,
                  "paddingLeft": 24,
                  "paddingRight": 24,
                  "position": "relative",
                  "textAlign": "left",
                  "textOverflow": "ellipsis",
                  "whiteSpace": "nowrap",
                  "width": "25%",
                }
              }
            >
              Account
            </th>
            <th
              className={undefined}
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              style={
                Object {
                  "backgroundColor": "inherit",
                  "color": "#9e9e9e",
                  "fontSize": "1.1em",
                  "fontWeight": "normal",
                  "height": 56,
                  "muiPrepared": true,
                  "paddingLeft": 24,
                  "paddingRight": 24,
                  "position": "relative",
                  "textAlign": "left",
                  "textOverflow": "ellipsis",
                  "whiteSpace": "nowrap",
                  "width": "20%",
                }
              }
            >
              Amount
            </th>
            <th
              className={undefined}
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              style={
                Object {
                  "backgroundColor": "inherit",
                  "color": "#9e9e9e",
                  "fontSize": "1.1em",
                  "fontWeight": "normal",
                  "height": 56,
                  "muiPrepared": true,
                  "paddingLeft": 24,
                  "paddingRight": 24,
                  "position": "relative",
                  "textAlign": "left",
                  "textOverflow": "ellipsis",
                  "whiteSpace": "nowrap",
                  "width": "20%",
                }
              }
            >
              Gen Time
            </th>
          </tr>
        </thead>
      </table>
    </div>
    <div
      style={
        Object {
          "height": "inherit",
          "muiPrepared": true,
          "overflowX": "hidden",
          "overflowY": "auto",
        }
      }
    >
      <table
        className={undefined}
        style={
          Object {
            "backgroundColor": "#ffffff",
            "borderCollapse": "collapse",
            "borderSpacing": 0,
            "fontFamily": "Roboto, sans-serif",
            "tableLayout": "fixed",
            "width": "100%",
          }
        }
      >
        <tbody
          style={
            Object {
              "muiPrepared": true,
              "textAlign": "center",
            }
          }
        >
          <tr
            className={undefined}
            style={
              Object {
                "borderBottom": false,
                "color": "rgba(0, 0, 0, 0.87)",
                "height": 48,
                "muiPrepared": true,
              }
            }
          >
            <td
              className={undefined}
              colSpan="5"
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              style={
                Object {
                  "backgroundColor": "inherit",
                  "fontSize": 13,
                  "height": 48,
                  "muiPrepared": true,
                  "overflow": "hidden",
                  "paddingLeft": 24,
                  "paddingRight": 24,
                  "textAlign": "left",
                  "textOverflow": "ellipsis",
                  "whiteSpace": "nowrap",
                }
              }
            >
              No Transactions to display
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;
